src/main/resources/assets/tfc/patchouli_books/book/en_us/entries/getting_started/predators.json

```diff
@@ -50,6 +50,48 @@
       "text": "The Panther hunts during the night.",
       "entity": "tfc:panthertfc",
       "scale": 0.7
+    },
+    {
+      "type": "entity",
+      "name": "Direwolf",
+      "text": "The Direwolf hunts during the night.",
+      "entity": "tfc:direwolftfc",
+      "scale": 0.7
+    },
+    {
+      "type": "entity",
+      "name": "Jackal",
+      "text": "The Jackal hunts during the night.",
+      "entity": "tfc:jackaltfc",
+      "scale": 0.7
+    },
+    {
+      "type": "entity",
+      "name": "Black Bear",
+      "text": "The Black Bear hunts during the day.",
+      "entity": "tfc:blackbeartfc",
+      "scale": 0.7
+    },
+    {
+      "type": "entity",
+      "name": "Cougar",
+      "text": "The Cougar hunts during the night.",
+      "entity": "tfc:cougartfc",
+      "scale": 0.7
+    },
+    {
+      "type": "entity",
+      "name": "Panther",
```

Comment:
Another panther. Maybe it should be something else, but seems only 5 predators were added.